webdriver-spec.html

```diff
@@ -3237,183 +3237,428 @@ <h2 id=element-displayedness>Element Displayedness</h2>
-                          </ol>
-                        </li>
-                        <li>Return the response.</li>
-                      </ol>
-                    </li>
-                  </ol>
-                </section>
-        </section>
-        <section>
-          <h2>Reporting Errors</h2>
-          <p></p>
-        </section>
+            </dd>
+
+            <dt>Otherwise:</dt>
+            <dd>
+              <ol>
+                <li><p>Let <var>result</var> be a
+                    new <code>Object</code> object.</p></li>
+                </ol>
+            </dd>
+          </dl>
+        </li>
+
+        <li><p>For each enumerable own property in <var>value</var>,
+        run the following substeps:</p>
+          <ol>
+            <li><p>Let <var>name</var> be the name of the property.</li>
+
+            <li><p>Let <var>source property value</var> be the result
+                of <a>getting a property</a> named <var>name</var>
+                from <var>value</var>. If doing so causes script to be
+                run, and that script throws an exception,
+                return <a>error</a> with code <a>javascript
+                error</a>.</li>
+
+            <li><p>Let <var>cloned property result</var> be the result of
+                calling the <var>clone algorithm</var> with
+                arguments <var>source property value</var>
+                and <var>seen</var>.</p></li>
+
+            <li><p>If <var>cloned property result</var> is
+                a <a>success</a>, <a>set a property</a>
+                of <a>result</a> with name <var>name</var> and value
+                equal to <var>cloned property result</var>'s data.</li>
+
+            <li><p>Otherwise, return <var>cloned property result</var>.</li>
+          </ol>
+        </li>
+  </ol>
+
+  <p>The steps for <dfn>extracting the script arguments from a request</dfn>,
+  are:</p>
+   <ol>
+      <li><p>If the <a>current browsing context</a> is <a>no longer
+          open</a>, return an <a>error</a> with code <a>no such
+          window</a>.</p></li>
+
+      <li><p>Let <var>script</var> be the result of getting a property
+          named <code>script</code> from the parameters argument.</p></li>
+
+      <li><p>If <var>script</var> is not a <code>String</code> return
+          an <a>error</a> with code <a>invalid argument</a>.</p></li>
+
+      <li><p>Let <var>args</var> be the result of getting a property
+          named <code>args</code> from the parameters argument.</p></li>
+
+      <li><p>If <var>args</var> is not an <code>Object</code> or
+          its <code>[[\Class]]</code> internal property is
+          not <code>Array</code> or <code>Object</code>,
+          return <a>error</a> with code <a>invalid
+          argument</a>.</p></li>
+
+       <li><p>Let <var>arguments</var> be a <a>List</a> consisting of
+          a <a>json deserialization</a> of each item
```

Comment:
It means a specific thing in the ES spec, which is what I need here. The link is missing though.